src/parse.rs

```diff
@@ -179,6 +181,7 @@ impl CloneRef for FilterType {
     fn clone_ref(&self, py: Python<'_>) -> Self {
         match self {
             Self::Add(arg) => Self::Add(arg.clone()),
+            Self::AddSlashes => Self::AddSlashes,
```

Comment:
I would test this, and the `PartialEq` and `PyEq` traits, by adding to the `test_filter_type_clone_ref` test right at the end of this file:

```rust
            let add_slashes = FilterType::AddSlashes;
            let cloned = add_slashes.clone_ref(py);
            assert_eq!(add_slashes, cloned);
            assert!(add_slashes.py_eq(&cloned, py));
```